packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -71,22 +73,53 @@ import {
   subscribeResourceSets,
 } from 'xo'
 
+@addSubscriptions(props => ({
+  resourceSet: cb =>
+    subscribeResourceSets(resourceSets =>
+      cb(find(resourceSets, { id: props.resourceSet }))
+    ),
+}))
+@connectStore(() => {
+  return (state, props) => ({
+    isAdmin: isAdmin(state, props),
+    resolvedResourceSet: getResolvedResourceSet(
+      state,
+      props,
+      !props.isAdmin && props.resourceSet !== undefined
```

Comment:
Don't forget to add the usual comment since it's still a workaround and we need to understand what the 3rd parameter is when we read the code.